### ISSUE TITLE
Error 500 backend

### ISSUE THREAD

**Issue by operatorone** [inspiredminds/contao-content-templates#2]
If I click on the "Template"-Link in the backend in the content section I get an error 500 and the following log entries:

`[2023-07-06T17:30:10.698065+02:00] request.INFO: Matched route "contao_backend". {"route":"contao_backend","route_parameters":{"_route":"contao_backend","_scope":"backend","_token_check":true,"_controller":"Contao\\CoreBundle\\Controller\\BackendController::mainAction"},"request_uri":"https://relaunch.ZZZZ.de/contao?act=edit&do=content_templates&id=1&ref=W_tVHCJn&rt=f496e099bf83713.LgNewbpaHeWiBVmfLmaBqYvHBocKEmUDvpb0BT6X9hI.RDo8s-Utcd3sczfxTwfn-tSORfVmdSd3h8-MRHXlv1YXchiI4iJys-dEFA&s2e=1&table=tl_content_template_article","method":"GET"} []`

`[2023-07-06T17:30:10.750527+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type' in 'field list'" at /html/contao_live/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 67 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 1054): An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type' in 'field list' at /html/contao_live/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:67)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1054): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type' in 'field list' at /html/contao_live/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'type' in 'field list' at /html/contao_live/vendor/doctrine/dbal/src/Driver/PDO/Statement.php:121)"} []`

It's running on a Contao 4.13.25

**Comment by fritzmg** [comment]
It looks like you did not update the database after installing this extension.

**Comment by operatorone** [comment]
Nope, the database is up-to-date. I made a small GIF to show when the error occurs.

https://www.dropbox.com/scl/fi/qljp6b90iw7cjhvie0kec/2023-07-07-12.50.35.gif?rlkey=xhqdtrxqoc1cx59p85i1oke9h&dl=0

**Comment by fritzmg** [comment]
Please do not use screenshots. Use the debug-mode and post (or analyse) the stack trace.

**Comment by fritzmg** [comment]
This issue occurs due to an incompatibility with [`erdmannfreunde/theme-toolbox`](https://github.com/erdmannfreunde/theme-toolbox). More specifically it occurs because `inspiredminds/contao-content-templates` copies the `tl_article` DCA for `tl_content_template_article` - and thus also copies any callbacks that will exist at that point, including the `tl_article` `onload` callback from `erdmannfreunde/theme-toolbox`. This needs to be fixed in `inspiredminds/contao-content-templates`.

As a workaround you can add the following in your Contao instance:

```php
// contao/dca/tl_content_template_article.php
$i = array_search([
    'ErdmannFreunde\ThemeToolboxBundle\EventListener\DataContainer\RegisterFieldsInPaletteListener',
    'onLoadContentCallback',
], $GLOBALS['TL_DCA']['tl_content_template_article']['config']['onload_callback'], true);

if (false !== $i) {
    unset($GLOBALS['TL_DCA']['tl_content_template_article']['config']['onload_callback'][$i]);
}
```

**Comment by fritzmg** [comment]
Should be fixed in `1.2.3`.

**Comment by operatorone** [comment]
Awesome. Thank you. Works like a charm.
